### ISSUE TITLE
engine line sorting

### ISSUE THREAD

**Issue by MartinRuz** [ml-research/liground#118]
sometimes the engine does not sort by best evaluation

**Comment by Zerthox** [comment]
I am fairly certain that we display the multipv lines in the order that engine puts them in.
According to UCI specification the engine should always send the best move as `multipv 1`.

From http://wbec-ridderkerk.nl/html/UCIProtocol.html:
```
* multipv 
	this for the multi pv mode.
	for the best move/pv add "multipv 1" in the string when you send the pv.
	in k-best mode always send all k variants in k strings together.
```

We could reorder them by cp but this should not be necessary unless the engine does not comply with the UCI specification.

**Comment by MartinRuz** [comment]
I agree, but anyway we had this problem repeatedly and it was also reported in the user study, so something must be going wrong somewhere
